runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/AnagramClue.java

```diff
@@ -71,85 +71,85 @@
-		new AnagramClue("RIP MAUL", "Primula", new WorldPoint(2454, 2853, 1), "Myth's Guild, first floor"),
-		new AnagramClue("SAND NUT", "Dunstan", new WorldPoint(2919, 3574, 0), "Anvil in north east Burthorpe", "How much smithing experience does one receive for smelting a blurite bar?", "8"),
-		new AnagramClue("SLAM DUSTER GRAIL", "Guildmaster Lars", new WorldPoint(1649, 3498, 0), "Woodcutting guild, Zeah"),
+		new AnagramClue("R SLICER", "Clerris", new WorldPoint(1761, 3850, 0), "Arceuus mine", "If I have 1,000 blood runes, and cast 131 ice barrage spells, how many blood runes do I have left?", "738"),
+		new AnagramClue("RIP MAUL", "Primula", new WorldPoint(2454, 2853, 1), "Myths' Guild, first floor"),
+		new AnagramClue("SAND NUT", "Dunstan", new WorldPoint(2919, 3574, 0), "Anvil, north-east Burthorpe", "How much smithing experience does one receive for smelting a blurite bar?", "8"),
+		new AnagramClue("SLAM DUSTER GRAIL", "Guildmaster Lars", new WorldPoint(1649, 3498, 0), "Woodcutting Guild, Hosidius"),
 		new AnagramClue("SLIDE WOMAN", "Wise Old Man", new WorldPoint(3088, 3253, 0), "Draynor Village", "How many bookcases are in the Wise Old Man's house?", "28"),
 		new AnagramClue("SNAKES SO I SAIL", "Lisse Isaakson", new WorldPoint(2351, 3801, 0), "Neitiznot", "How many arctic logs are required to make a large fremennik round shield?", "2"),
 		new AnagramClue("TAMED ROCKS", "Dockmaster", new WorldPoint(1822, 3739, 0), "Port Piscarilius, NE of General store", "What is the cube root of 125?", "5"),
-		new AnagramClue("TEN WIGS ON", "Wingstone", new WorldPoint(3389, 2877, 0), "Between Nardah & Agility Pyramid"),
-		new AnagramClue("THICKNO", "Hickton", new WorldPoint(2822, 3442, 0), "Catherby fletching shop", "How many ranges are there in Catherby?", "2"),
+		new AnagramClue("TEN WIGS ON", "Wingstone", new WorldPoint(3389, 2877, 0), "North of Agility Pyramid"),
+		new AnagramClue("THICKNO", "Hickton", new WorldPoint(2822, 3442, 0), "Catherby ranging shop", "How many ranges are there in Catherby?", "2"),
 		new AnagramClue("TWENTY CURE IRON", "New Recruit Tony", new WorldPoint(1503, 3553, 0), "Shayzien Graveyard"),
-		new AnagramClue("UNLEASH NIGHT MIST", "Sigli the Huntsman", new WorldPoint(2660, 3654, 0), "Rellekka", "What is the combined slayer requirement of every monster in the slayer cave?", "302"),
-		new AnagramClue("VESTE", "Steve", new WorldPoint(2432, 3423, 0), "Upstairs Wyvern Area or Stronghold Slayer Cave", "How many farming patches are there in Gnome stronghold?", "2"),
+		new AnagramClue("UNLEASH NIGHT MIST", "Sigli the Huntsman", new WorldPoint(2660, 3654, 0), "Rellekka entrance", "What is the combined slayer requirement of every monster in the slayer cave?", "302"),
+		new AnagramClue("VESTE", "Steve", new WorldPoint(2432, 3423, 0), "Upstairs Skeletal Wyvern area in Asgarnia or Stronghold Slayer Cave (after Monkey Madness 2)", "How many farming patches are there in Gnome stronghold?", "2"),
```

Comment:
There's only one skeletal wyvern area, no need to specify further than that.
```suggestion
		new AnagramClue("VESTE", "Steve", new WorldPoint(2432, 3423, 0), "Upstairs Skeletal Wyvern area or Stronghold Slayer Cave (after Monkey Madness 2)", "How many farming patches are there in Gnome stronghold?", "2"),
```